src/jvmTest/kotlin/mu/KotlinLoggingTest.kt

```diff
@@ -1,18 +1,20 @@
 package mu
 
+import mu.KotlinLogging.logger
 import org.junit.Assert.assertEquals
 import org.junit.Test
 import org.slf4j.LoggerFactory
 
-private val logger = KotlinLogging.logger { }
-private val loggerFromSlf4j = KotlinLogging.logger(LoggerFactory.getLogger("mu.slf4jLogger"))
+private val logger = logger { }
+private val loggerFromSlf4j = logger(LoggerFactory.getLogger("mu.slf4jLogger"))
 private val loggerFromSlf4jExtension = LoggerFactory.getLogger("mu.slf4jLoggerExtension").toKLogger()
 
```

Comment:
can we have a test also when logger() is defined outside of the class?

Reply:
I don't think it's possible to use an reified inline extension function at the top level - the compiler only recognizes the other three public `KotlinLogging.logger()` functions as valid. This declaration works:
```
private val inlineLogger = KotlinLogging.logger()
```
However, this is declaring a logger for `mu.KotlinLogging`, not for the class containing the top-level variables.